packages/scss/src/components/inputs/field/_field.scss

```diff
@@ -56,8 +56,8 @@
 		/* Display */
 		&.mod-inline {
 			display: inline-flex;
-			margin-left: _component("field.input.inline-margin");
-			margin-right: _component("field.input.inline-margin");
+			// margin-left: _component("field.input.inline-margin");
+			margin-right: _component("field.input.inline-margin") * 2;
```

Comment:
Could you remove the comment ? :) 
And I think it would be more clear if we double the value inside the theming file